pnpm-lock.yaml

```diff
@@ -7047,6 +7044,9 @@ packages:
   zod@3.24.4:
```

Comment:
Old zod and zod-to-json-schema wasn't removed in v18 because v18 uses eslint 9.26.0 which has a dep (`@modelcontextprotocol/sdk`) that depends on `zod-to-json-schema`. eslint 9.34.0 on master no longer seems to load zod somewhere down the tree.